docassemble/ALWeaver/data/sources/question_library_patterns.yml

```diff
@@ -0,0 +1,197 @@
+# Words that are reserved exactly as they are
+whole words:

```

Comment:
This file is neat, a bit better than the current python approach, but I don't see it being used anywhere? It doesn't seem necessary to the review screen changes, can it be split off into another PR?